### ISSUE TITLE
Focus on navigation buttons not visible 

### ISSUE THREAD

**Issue by chmst** [joomla/joomla-cms#38135]
### Steps to reproduce the issue
1. Enable content page navigation plugin for your blog (is default)
2. Oben an article in a blog (in Cassiopeia)
3. See the navigation Button(s), set the focus on one of them

### Expected result
Navigation Buttons show a visible focus ring when they get the focus.

### Actual result
No focus border. 

### Device and system info

* What version of Joomla were you using?
4.1

* What device were you using? (eg, phone, laptop)
laptop

* What web browser were you using? (eg, Chrome, Safari)
Chrome, Firefox

* If you were using assistive technology, please let us know the device or software you were using (eg VoiceOver, JAWS, sip-and-puff device, etc)

### Additional comments

If you know which [WCAG](https://www.w3.org/TR/WCAG/) or [ATAG](https://www.w3.org/TR/ATAG20/) criteria this violates, please indicate it here:
https://www.w3.org/WAI/WCAG21/Understanding/focus-visible.html

**Comment by chmst** [comment]
Closing - please test https://github.com/joomla/joomla-cms/pull/38140

**Comment by richard67** [comment]
Opening again since the PR has been closed

**Comment by alikon** [comment]
plese test #38236
